.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: ''
+assignees: guzongmin
+
+---
+
+<!--- Provide a general summary of the issue in the Title above -->
+**Describe the bug**
```

Comment:
Instead of formatting the title as below,
````
**Describe the bug**
````
I think it is more appropriate to format it as the title of a section
````
# Describe the bug
````
This way, the submitter of the issue can have subsections or subsubsections to make the text look more structured.